kits/scale-pilot/prompts/scale-pilot_llmnode-558_user_4.md

```
@@ -0,0 +1,18 @@
+You are provided with the parsed architecture details.
```

markdownlint-cli2 (0.23.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)